components/select/src/auro-select.js

```diff
@@ -545,11 +566,13 @@ export class AuroSelect extends LitElement {
           chevron
           part="dropdown">
           <span slot="trigger" aria-haspopup="true" id="triggerFocus">
-            ${this.value ? this.displayValue : html`<span class="placeholder">${this.placeholder}</span>`}
+            ${this.value ? this.displayValue : html`<span class="placeholder">${this.placeholder || this.labelText}</span>`}
```

Comment:
I think this is problematic for two reasons.

You have a span that you have identified as "placeholder" but you are also conditionally putting non-placeholder content into it. That's an anti-pattern we should avoid.

Also, the label for form elements shouldn't change as it will notably impact the experience a voice over utility relies on.

I think you are safer having two different elements that you simply hide/show the one you need based on state.

Reply:
Also, I think we should be keeping slot content out of properties.